### DISCUSSION TITLE
Hello devs help!

### DISCUSSION THREAD

**Discussion by ghost** [community/community#58817] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

could i use the copilot when i just buy sub
do i need to wait for whitelist or can i use instantly when i buy?

**Comment by davecheney** [comment, marked as answer]
@suygetsu1 there is no waitlist for Copilot. You can sign up to start a free trial here, https://github.com/features/copilot/

**Reply by ghost** [reply]
wait so you are saying i can buy copilot now and use it on visual studio code right?
but there is a wait list for visual studio 2022

**Reply by davecheney** [reply]
There is no wait list for visual studio 2022. You can use copilot on any of our supported editors.

**Reply by ghost** [reply]
oh i understand actually im trying to use copilot chat not the write comment and press tab thing

what do you think when it will be open for everyone that buy it requires wait list.
